frontend/src/services/Controller.ts

```diff
@@ -116,6 +116,7 @@ function getEventHandlers() {
       await store.dispatch.licensing.fetch()
       await store.dispatch.accounts.init()
       await store.dispatch.devices.init()
+      await store.dispatch.ui.init()
```

Comment:
@JamieRuderman could we use an await promise all ?

Reply:
@mauricio0000 if we did Promise.all it would resolve all the promises asynchronously... I think it's safer to do them synchronously as above.